extensions/src/doodlebot/Doodlebot.ts

```diff
@@ -645,83 +645,202 @@ export default class Doodlebot {
     line;
     lineCounter = 0;
     detector;
-
+    j = 0;
     async followLine() {
         let first = true;
         const delay = 0.5;
```

Comment:
would you able to comment what each of these numbers mean? 
`const interval = 350; // 1/15th of a second` is perfect